libs/vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -991,6 +991,11 @@ class Joke(BaseModel):
         The format of the dictionary should follow Open API schema.
     """
 
+    cached_content: Optional[str] = None
```

Comment:
should we add it is a kwarg to the `_generate` method too?

and maybe add a note to docsting on how to use caching.

Reply:
I'll do it!